language/nodejs/build-images.md

```diff
@@ -220,7 +220,6 @@ To list images, simply run the `images` command.
 $ docker images
 REPOSITORY          TAG                 IMAGE ID            CREATED              SIZE
 node-docker         latest              3809733582bc        About a minute ago   945MB
-node                12.18.1             f5be1883c8e0        2 months ago         918MB
 ```
 
 You should see at least two images listed. One for the base image `node:12.18.1` and the other for our image we just build `node-docker:latest`.
```

Comment:
This would need to be updated / rephrased as well.

I think the discrepancy is probably that the example was written with the classic builder. When using BuildKit as builder, the base-image will not appear in the list of local images.


If we want to rewrite this assuming BuildKit is used, probably we should update the build-progress output as well, as it's currently shows the output of the classic builder;

```
Sending build context to Docker daemon  82.94kB
Step 1/7 : FROM node:12.18.1
---> f5be1883c8e0
Step 2/7 : WORKDIR /code
...
Successfully built e03018e56163
Successfully tagged node-docker:latest
```

Reply:
@thaJeztah @crazy-max I've started a draft PR here to display BuildKit output - https://github.com/docker/docker.github.io/pull/13221.